go/billing/tests/helpers.py

```diff
@@ -74,7 +93,7 @@ def mk_transaction(account, tag_pool_name='pool1',
 
 def mk_statement(account,
                  title=settings.MONTHLY_STATEMENT_TITLE,
-                 type=Statement.TYPE_MONTHLY,
+                 _type=Statement.TYPE_MONTHLY,
```

Comment:
I always think of leading underscores as meaning that something is unused or an implementation detail. Perhaps ``stmt_type`` would be better?